addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -93,7 +93,7 @@
 <template name="Course Sidebar (infos, CTA)" id="course_join" inherit_id="website_slides.course_join">
     <!-- Channel main template: override button to join channel -->
     <xpath expr="//div[hasclass('o_wslides_js_course_join')]" position="inside">
-        <t t-if="(not channel.is_member or channel.can_publish) and channel.enroll == 'payment'">
+        <t t-elif="(not channel.is_member or channel.can_publish) and channel.enroll == 'payment'">
```

Comment:
Sorry, maybe my previous comment was not clear enough, but I would rather that we keep a simple "if" here that re-test everything needed.

Adding a t-elif "inside" a div is very much dependent on the base template, and it's a bit hard to follow when you get here as you have to go inside the other module to see in which case this block is shown.